### ISSUE TITLE
Next steps: can't click on tools at the bottom of the list

### ISSUE THREAD

**Issue by julie-sullivan** [alexkalderimis/staircase#67]
tools at the bottom of the "next steps" tools list are not clickable, page height issue

**Comment by joshkh** [comment]
In the event of a categorical approach, we should consider using task, action, or data oriented labels.

Examples:
Visualize
Export
Enrich
Save
Convert
Transform
Search

**Comment by julie-sullivan** [comment]
@joshkh NO! That's the *opposite* of we want to do. We want a data-centered approach, in the next steps, on the front page, everywhere.
